sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/security/CertificateNamesGenerator.java

```diff
@@ -0,0 +1,50 @@
+package com.mesosphere.sdk.offer.evaluate.security;
+
+import org.bouncycastle.asn1.x500.X500Name;
+import org.bouncycastle.asn1.x509.GeneralName;
+import org.bouncycastle.asn1.x509.GeneralNames;
+
+/**
+ * A {@link CertificateNamesGenerator} creates relevant names for given service pod.
+ *
+ * // TODO(mh): What about IP per container feature??
```

Comment:
Yes, first pass should just be DNS (IIRC we discussed this in slack). Obviously not 100% ideal, but best we can presently do.